app/views/templates/Layout.scala.html

```diff
@@ -78,24 +78,26 @@
       @scriptsBlock.getOrElse(Html(""))
     }
 
-    @beforeContentBlock = {
-      <div class="govuk-!-display-none-print">
-      @govukPhaseBanner(PhaseBanner(
-        tag = Some(Tag(
-          content = Text(Messages.ServicePhase.beta.show)
-        )),
-        content = HtmlContent(Messages.ServicePhase.bannerText(appConfig.Urls.betaFeedbackUrl(request)).show)
-      ))
-      @if(appConfig.welshLanguageSupportEnabled) {
-        @languageSelection()
-      }
-      @{
-        backUrl map { url =>
-          govukBackLink(BackLink(href = url, attributes = Map("id" -> "back")))
+@beforeContentBlock = {
+    <div class="govuk-!-display-none-print">
+        @govukPhaseBanner(PhaseBanner(
+            tag = Some(Tag(
+                content = Text(Messages.ServicePhase.beta.show)
+            )),
+            content = HtmlContent(Messages.ServicePhase.bannerText(appConfig.Urls.betaFeedbackUrl(request)).show)
+        ))
+        <div class="language-toggle-position">
```

Comment:
this extra div with custom css is not required as it goes against the principle of making the toggle appear above the back link